src/scripts/salasana.js

```diff
@@ -0,0 +1,46 @@
+import {startProcess, stopProcess} from '/shared/scripts/progressbar.js';
+import {eventHandled} from '/shared/scripts/uiUtils.js';
+import {showSnackbar} from '/shared/scripts/snackbar.js';
+import {changePassword} from "/scripts/callRest.js";
+
+window.initializeVaihto = function () {
```

Comment:
Should the name for the function be initializeSalasana or initializeVaihto?

Reply:
Fixed